<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="9" column="7">
     <widget class="QLabel" name="vehicleLabel">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="6" column="7">
     <widget class="QLabel" name="styleLabel">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="11" column="7">
     <widget class="QLabel" name="ratingLabel">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="4" column="7">
     <widget class="QPushButton" name="AddRowButton">
      <property name="text">
       <string>Add Row</string>
      </property>
     </widget>
    </item>
    <item row="7" column="7">
     <widget class="QLabel" name="rangeLabel">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="1" column="7">
     <widget class="QPushButton" name="PBLoadFromFile">
      <property name="text">
       <string>Load Data From a File</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="0" rowspan="8">
     <widget class="QTableView" name="tableView"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
        <italic>false</italic>
        <bold>true</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>New Car models â€“ India</string>
      </property>
     </widget>
    </item>
    <item row="2" column="7">
     <widget class="QPushButton" name="deleteButton">
      <property name="text">
       <string>Delete Row</string>
      </property>
     </widget>
    </item>
    <item row="10" column="7">
     <widget class="QLabel" name="priceLabel">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="5" column="7">
     <widget class="QLabel" name="carLabel">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="8" column="7">
     <widget class="QLabel" name="transmissionLabel">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QPushButton" name="saveButton">
      <property name="text">
       <string>Save Data To a File</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Car:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Style:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Range:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Transmission:</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Vehicle:</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Price:</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Rating:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuStudent">
    <property name="title">
     <string>Student Info</string>
    </property>
    <addaction name="actionStudent"/>
   </widget>
   <addaction name="menuStudent"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionStudent">
   <property name="text">
    <string>Information</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
